sphinx/util/i18n.py

```diff
@@ -91,8 +91,11 @@ def locale_dirs(self) -> Generator[str, None, None]:
 
         for locale_dir in self._locale_dirs:
             locale_dir = path.join(self.basedir, locale_dir)
-            if path.exists(path.join(locale_dir, self.language, 'LC_MESSAGES')):
+            locale_path = path.join(locale_dir, self.language, 'LC_MESSAGES')
+            if path.exists(locale_path):
                 yield locale_dir
+            else:
+                logger.info(__('locale_dir %s does not exists'), locale_path)
```

Comment:
Could you use `logger.warning()` instead here. I believe it's a wrong entry. So it should be fixed by user's side. So using a warning is better for this case.

Reply:
I did put a warning at first but I was wondering if it wasn't making false negative. Is it a valid scenario where you have multiple `locale_dirs` but only some containing the translations? Or not having all languages in all `locale_dirs`

Reply:
Exactly. It's not good to use the WARNING level for this purpose. From its perspective, the INFO is also noisy. How about using VERBOSE instead? Then the message will be shown only `-v` option is given. I think it's better.

Reply:
Good enough for me :)